students/piotrowski_stanislaw/lesson_01_basics/L1_IFN_04.py

```diff
@@ -0,0 +1,13 @@
+#https://snakify.org/lessons/integer_float_numbers/problems/fractional_part/
+#piotrsta
+
+import math
+
+print('Podaj dodatnia liczbe rzeczywista: ')
+num = float(input())
+
+frac, whole = math.modf(num)
```

Comment:
poprawne rozwiązanie, można tez było użyć działań matematycznych żeby to osiągnąć.

Reply:
Poprawiłem na działania matematyczne.